api/turing/api/ensembling_job_api_test.go

```diff
@@ -385,3 +385,205 @@ func TestEnsemblingJobController_CreateEnsemblingJob(t *testing.T) {
 		})
 	}
 }
+
+func TestEnsemblingJobController_GetEnsemblingJob(t *testing.T) {
```

Comment:
It would be good to add a test case with more than 1 total pages and page number > 1,